app/js/components/Error.jsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
 
-
-const Error = ({ error }) => (
-    <div className="page-header">
-        <h1>{error}</h1>
+const Error = ({ children }) => (
```

Comment:
I hadn't noticed this before, but I like how these parens are styled as braces when the function returns a single JSX expression.

:+1: